config/modules/dts.conf

```diff
@@ -0,0 +1,12 @@
+# Baseline test file for officially included tests.
+
+# Tests to build.
+build ${DAQ_LIB}/docker/modules
+
+# Default built-in tests.
+add pass first
+add fail first
+add ping first
+
+# Additional base modules
+include ${DAQ_LIB}/subset/pentests/build.conf
```

Comment:
Doesn't this also include the discover test which is in a docker container?

Reply:
I've been using 
<pre>
add pass
add fail
add ping
add nmap
</pre>

Reply:
fixed